### PR TITLE
fix(synthesizers): added missing model name

### PR DESCRIPTION
**Pull request by mglcampos**
- Added model name to TimeGAN and BaseModel.

### CODE REVIEW

File: src/ydata_synthetic/synthesizers/gan.py [hunk 1]
```diff
@@ -22,6 +22,9 @@
 TrainParameters = namedtuple('TrainParameters', _train_parameters, defaults=('', None, 300, 50, None))
 
 class BaseModel():
+
+    __MODEL__='BaseModel'
```
Comment (fabclmnt) [inline src/ydata_synthetic/synthesizers/gan.py:26]: I would either set this property no None or not have it on the BaseModel.

**Review by fabclmnt — CHANGES_REQUESTED** [review]
__MODEL__ value in the BaseModel class

File: src/ydata_synthetic/synthesizers/gan.py (hunk 1 above)
Comment (mglcampos) [reply src/ydata_synthetic/synthesizers/gan.py:26]: Between validating the existence of "__Model__" or setting it everywhere I have chosen the latter. 
But I understand your point, I'll remove it.

Comment (fabclmnt) [reply src/ydata_synthetic/synthesizers/gan.py:26]: if we keep it as '' that works same. Without the need for validations I think

Comment (mglcampos) [reply src/ydata_synthetic/synthesizers/gan.py]: yes it works

**Pull request closed without merging**